sources/integrations/integrations.cpp

```diff
@@ -186,7 +186,10 @@ void Integrations::add(const QVariantMap& config, QObject* obj, const QString& t
     m_integrationsFriendlyNames.insert(id, config.value(Config::KEY_FRIENDLYNAME).toString());
     m_integrationsTypes.insert(id, type);
     IntegrationInterface* ii = qobject_cast<IntegrationInterface*>(obj);
-    ii->connect();
+
+    // FIXME: connect to entity loading done signal and then connect
```

Comment:
Is there already a GitHub issue for that? If not, please create issue and reference it in the FIXME